_delphi_utils_python/delphi_utils/weekday.py

```diff
@@ -12,11 +12,13 @@ class Weekday:
     """Class to handle weekday effects."""
 
     @staticmethod
-    def get_params(data, denominator_col, numerator_cols, date_col, scales, logger):
+    def get_params(data, denominator_col, numerator_cols, date_col, scales, logger, solver=cp.CLARABEL):
```

Comment:
this format will let us keep the actual engine hidden from callers.
then `claims_hosp/delphi_claims_hosp/update_indicator.py` can be changed to use `Weekday.get_params_legacy()` instead.
and then we should add a test for `get_params_legacy()` that includes the values replaced in [the previous Clarabel/ECOS PR](https://github.com/cmu-delphi/covidcast-indicators/pull/1966/files).

```suggestion
    def get_params_legacy(data, denominator_col, numerator_cols, date_col, scales, logger):
        # This preserves the older default behavior of using the ECOS solver.
        # NOTE: "ECOS" solver will not be installed by default as of cvxpy 1.6
        return get_params(data, denominator_col, numerator_cols, date_col, scales, logger, solver_override=cp.ECOS)

    @staticmethod
    def get_params(data, denominator_col, numerator_cols, date_col, scales, logger, solver_override=None):
```